pom.xml

```diff
@@ -69,7 +69,13 @@
         <dependency>
             <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>6.2.0.CR1</version>
+            <version>7.0.3.Final</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
```

Comment:
The server provides `jakarta.validation:jakarta.validation-api:3.0.0`. I think we should that one?

Reply:
Thanks, fixed.